pkg/build/api/types.go

```diff
@@ -19,9 +19,6 @@ type Build struct {
 	// A human readable message indicating details about why the build has this status
 	Message string `json:"message,omitempty"`
 
-	// PodName is the name of the pod that is used to execute the build
-	PodName string `json:"podName,omitempty"`
```

Comment:
the downside to doing this is we can now never change the GetBuildPodName function since the only coupling between builds and their pods is through that function, regardless of when the build was created.  What was the motivation to remove this field?